sample/peer/cmd/request.go

```diff
@@ -65,9 +68,20 @@ type clientStack struct {
 }
 
 func request(client client.Client, arg string) {
-	req := []byte(arg)
-	res := <-client.Request(req)
-	fmt.Println("Reply:", string(res))
+	var timeoutChan <-chan time.Time
+	timeout := viper.GetDuration("client.timeout")
+
+	if timeout > 0 {
+		timeoutChan = time.After(timeout)
+	}
+
+	select {
+	case res := <-client.Request([]byte(arg)):
+		fmt.Println("Reply:", string(res))
+	case <-timeoutChan:
+		fmt.Println("Client Request timer expired.")
+		os.Exit(1)
```

Comment:
[Initially, `request` function would return an error](https://github.com/hyperledger-labs/minbft/blob/50d684f2aff06919cbcf40287937a6ef55140063/sample/peer/cmd/request.go#L70) that is propagated by `cobra` up the call stack, and finally reported [here](https://github.com/hyperledger-labs/minbft/blob/50d684f2aff06919cbcf40287937a6ef55140063/sample/peer/cmd/root.go#L48). Maybe we should restore that behavior (in a separate PR)? 

In that case, it seems not necessary to output the error here, because `sample/peer/cmd.Execute` will do that. We might also want to report the error to stderr.